heft-plugins/heft-lint-plugin/src/Eslint.ts

```diff
@@ -227,6 +229,21 @@ export class Eslint extends LinterBase<TEslint.ESLint.LintResult> {
         }
       }
     }
+
+    const sarifLogPath: string | undefined = this._sarifLogPath;
+    if (sarifLogPath) {
+      const { formatAsSARIF } = await import('./SarifFormatter');
+      const sarifString: string = JSON.stringify(
+        formatAsSARIF(lintResults, {
```

Comment:
This will include fixed issues as well. Intended?

Reply:
Yes